DOCUMENTATION.md

```diff
@@ -181,6 +181,27 @@ email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 If you omit this argument, the client will set your default email reply-to address for the notification.
 
+### Attach a document
+
+You can attach a document to an email notification.
+
+Contact the GOV.UK Notify team on the support page or through the Slack channel to enable this function for your service.
+
+The document must be a PDF file smaller than 2MB. To attach the document, you must:
+
+1. Define the PDF as a file object.
+1. Pass the file object as a value into the personalisation argument. For example:
+
+        ```python
+        with open('file.pdf', 'rb') as DOCUMENT_NAME:
```

Comment:
`f` is a conventional variable name for this. DOCUMENT_NAME is a bit misleading (since the value isn't actually a name) and it shouldn't be capitalize since it implies it's a global constant (which it isn't).